doc/vaulted-env.1.md

```diff
@@ -10,7 +10,8 @@ SYNOPSIS
 --------
 
 `vaulted env` *name* [*OPTIONS*]  
-`vaulted env --assume` *arn* [*OPTIONS*]
+`vaulted env --assume` *arn* [*OPTIONS*]  
+`vaulted env --refresh` *name* [*OPTIONS*]
```

Comment:
We probably don't need to call out each individual option (especially since this can be combined with things like `--assume`), `--assume` is just called out specifically because it is nearly its own feature. :)